src/main.rs

```diff
@@ -0,0 +1,32 @@
+mod build;
+
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+use anyhow::Result;
+use xshell::pushd;
+
+fn main() -> Result<()> {
+    let _d = pushd(project_root())?;
+    let flags = build::flags::Xtask::from_env()?;
+    match flags.subcommand {
+        build::flags::XtaskCmd::Help(_) => {
+            println!("{}", build::flags::Xtask::HELP);
+            Ok(())
+        }
+        build::flags::XtaskCmd::Dist(cmd) => cmd.run(),
+        build::flags::XtaskCmd::Napi(cmd) => cmd.run(),
+    }
+}
+
+fn project_root() -> PathBuf {
+    Path::new(
```

Comment:
We cannot assume that xtask is invoked via a cargo alias anymore, so perhaps it's better to require the user to take the cargo workspace path in?